_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(9); };
+    }
+
+    .sm-post-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(10); };
+    }
+
+    .sm-post-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include post(11); };
+    }
+
+
+    // CASE: medium screen bp
+    .md-col-1 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(1); };
+    }
+
+    .md-col-2 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(2); };
+    }
+
+    .md-col-3 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(3); };
+    }
+
+    .md-col-4 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(4); };
+    }
+
+    .md-col-5 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(5); };
+    }
+
+    .md-col-6 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(6); };
+    }
+
+    .md-col-7 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(7); };
+    }
+
+    .md-col-8 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(8); };
+    }
+
+    .md-col-9 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(9); };
+    }
+
+    .md-col-10 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(10); };
+    }
+
+    .md-col-11 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(11); };
+    }
+
+    .md-col-12 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .md-pre-1 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(1); };
+    }
+
+    .md-pre-2 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(2); };
+    }
+
+    .md-pre-3 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(3); };
+    }
+
+    .md-pre-4 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(4); };
+    }
+
+    .md-pre-5 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(5); };
+    }
+
+    .md-pre-6 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(6); };
+    }
+
+    .md-pre-7 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(7); };
+    }
+
+    .md-pre-8 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(8); };
+    }
+
+    .md-pre-9 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(9); };
+    }
+
+    .md-pre-10 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(10); };
+    }
+
+    .md-pre-11 {
+        @include susy-breakpoint($bp-screen-md, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .md-post-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.